### PR TITLE
impl: Add Luca Bandini as maintainer

### PR DESCRIPTION
**Pull request by Vombato**
As talked about in last SLSA Specification meeting I am proposing as maintainer for the project.

✍🏻 Also fixed the alphabetical order of the current maintainers, which was off.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -4,10 +4,11 @@
 
 | Name | Email | OpenSSF Slack | GitHub | Affilitaion |
 | --- | --- | --- | --- | --- |
-| Kris K | kkris@google.com | @Kris K | [kpk47](https://github.com/kpk47) | Google |
-| Michael Lieberman | mlieberman85@gmail.com | @Michael Lieberman | [mlieberman85](https://github.com/mlieberman85) | Kusari |
 | Joshua Lock | joshuagloe@gmail.com | @Joshua Lock |  [joshuagl](https://github.com/joshuagl) | Verizon |
+| Kris K | kkris@google.com | @Kris K | [kpk47](https://github.com/kpk47) | Google |
+| Luca Bandini | luca.vombato@gmail.com | @Luca Bandini | [Vombato](https://github.com/Vombato) | SIGHUP |
 | Mark Lodato | lodato@google.com |  @Mark Lodato | [MarkLodato](https://github.com/MarkLodato) | Google |
```
Comment (jkjell) [inline MAINTAINERS.md:10]: Looks like it was alphabetically ordered by last names before. Not sure if that was intentional.

Comment (kpk47) [reply MAINTAINERS.md:10]: It was. I usually sort people by last name, but don't feel strongly about it.

Comment (Vombato) [reply MAINTAINERS.md:10]: Makes sense, my mistake.
I'll fix it right away.

**Comment by Vombato** [comment]
Matching the new criteria proposed [here](https://github.com/slsa-framework/governance/pull/22), I withdraw my Maintainer proposal for the time being.
Will maybe re-apply once I gained some more experience and reputation on the project 😁

**Pull request closed without merging**